src/app/app.module.ts

```diff
@@ -236,6 +238,7 @@ const DEFAULT_SWIPER_CONFIG: SwiperConfigInterface = {
     DeleteFavoriteComponent,
     UserSettingComponent,
     AddCommentComponent,
+    NewSignUpComponent
```

Comment:
Please move it to the auth module(entry components)

Reply:
fixed